package.json

```diff
@@ -18,8 +18,10 @@
     "@types/node": "^8.0.20",
     "@types/react": "^16.0.2",
     "@types/react-dom": "^15.5.2",
+    "@types/react-router-dom": "^4.0.7",
```

Comment:
I found this article interesting: https://blog.mariusschulz.com/2017/01/13/typescript-vs-flow